src/modules/helpchan.ts

```diff
@@ -81,13 +84,68 @@ export class HelpChanModule extends Module {
 		return `${this.CHANNEL_PREFIX}${decidedChannel}`;
 	}
 
+	private getOngoingChannels() {
+		return this.client.channels.cache
+			.filter(
+				channel =>
+					(channel as TextChannel).parentID === categories.ongoing,
+			)
+			.array() as TextChannel[];
+	}
+
 	@listener({ event: 'ready' })
 	async startDormantLoop() {
 		setInterval(() => {
 			this.checkDormantPossibilities();
 		}, dormantChannelLoop);
 	}
 
+	@listener({ event: 'ready' })
+	async initialCheckEmptyOngoing() {
+		for (const channel of this.getOngoingChannels()) {
+			if (await this.checkEmptyOngoing(channel)) {
+				await this.startEmptyTimeout(channel);
+			}
+		}
+	}
+
+	// Utility function used to check if there are no messages in an ongoing channel, meaning the bot
+	// is the most recent message. This will be caused if somebody deletes their message after they
+	// claim a channel.
+	async checkEmptyOngoing(channel: TextChannel) {
+		const messages = await channel.messages.fetch();
```

Comment:
If you're worried about rate-limiting, I'd recommend moving this empty channel check to after the timeout, to debounce the `channel.messages.fetch()`.